changes/385.feature.md

```diff
@@ -0,0 +1 @@
+Add a new demo example to show various features of aiomonitor, especially using the GUI (also for PyCon APAC 2023 talk)
```

Comment:
I recommend using the past tense or some other way to describe end-user facing changes compared to the previous release. This is because the end users are a different audience compared to the contributors who would read Git commits.
Here's an example: https://yarl.aio-libs.org/en/stable/changes/#id1.

Reply:
The present tense and imperative mood is usually needed for commit messages. Git commits have different audience — their consumers are contributors/devs. So the commits have different considerations in terms of wording, the level of detail and angles of explanation..